readme.md

```diff
@@ -140,7 +168,13 @@ Instantiate an instructor named 'Andy' and call its `receivePresent` method with
 
 ```js
 // Your code starts here...
+function Instructor (name) {
+  this.name = name;
 
+  this.receivePresent = function (gift){
+    console.log(this.name + 'promptly drops the ' + this.gift + 'on the floor.')
```

Comment:
Close, but 2 things:
* `gift` is a local variable, not a property, so you can't put `this` in front of it (but you're right to do so for `name`).
* the receivePresent method would technically be ok (except for above), but it's better practice to put instance methods on the prototype, to prevent creating multiple copies of the method function.